ASPnetCoreMVC/Controllers/ProductController.cs

```diff
@@ -29,7 +29,8 @@ public IActionResult LoadProductTable(string name, string priceFilter, string so
                 ID = p.ID,
                 Name = p.Name,
                 Price = p.Price,
-                Stock = p.Stock
+                Stock = p.Stock,
+                CategoryName = p.Category.Name
```

Comment:
Product may not have a category, please check if the category is empty
(suggest use ? to check)